packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -926,10 +926,10 @@ export async function getAvailableSlots({ input, ctx }: GetScheduleOptions): Pro
           askedContactOwner: contactOwnerEmailFromInput,
           // One that we used as per Routing
           usedContactOwner: contactOwnerEmail,
-          routedHosts: routedHostsWithContactOwnerAndFixedHosts.map((host) => {
```

Comment:
Use the correct variable. `usersWithCredentials` is  the users we consider for slots.